### PR TITLE
feat: bump deps

### PR DESCRIPTION
**Pull request by jquense**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
-      "functions": "never" // Need to support editors' old versions of Node.
-    }]
+    "comma-dangle": [
+      "error",
+      {
+        "arrays": "always-multiline",
+        "objects": "always-multiline",
+        "imports": "always-multiline",
+        "exports": "always-multiline",
+        "functions": "never" // Need to support editors' old versions of Node.
```
Comment (taion) [inline .eslintrc]: can we drop this?

File: packages/eslint-config-4catalyzer/rules.js [hunk 2]
```diff
@@ -1,37 +1,57 @@
...
+    {
+      vars: 'all',
+      varsIgnorePattern: '^_',
+      args: 'after-used',
+      ignoreRestSiblings: false,
+      argsIgnorePattern: '^_',
+    },
+  ],
+  // allow `let foo, bar;`, but not `let foo = 1, bar ='hi';`
+  'one-var': ['error', { initialized: 'never' }],
```
Comment (taion) [inline packages/eslint-config-4catalyzer/rules.js]: :+1:

File: .eslintrc (hunk 1 above)
Comment (jquense) [reply .eslintrc]: i'm not sure yet...

Comment (jquense) [reply .eslintrc]: looks like yes for vscode

**Pull request merged**